src/Entities/DNSRecordType.php

```diff
@@ -73,8 +76,15 @@ public function __toString(): string
         return $this->type;
     }
 
+    /**
+     * @throws \RemotelyLiving\PHPDNS\Exceptions\InvalidArgumentException
+     */
     public static function createFromInt(int $code) : DNSRecordType
     {
+        if (!isset(self::CODE_TYPE_MAP[$code])) {
```

Comment:
This will throw an Exception instead of a `Notice: Undefined offset:` so my application will still break.

Reply:
Right. Are you calling this factory method directly in your app? I'm wary of `createOrReturnNull` type of stuff as it means a lot of `null` case handling down the line. If you're just using the Resolvers they ignore invalid argument exceptions like here https://github.com/remotelyliving/php-dns/blob/master/src/Resolvers/ResolverAbstract.php#L114

Reply:
You could always `try / catch` if you are calling the factory method directly and handle the specifics there if you like. I guess the better question is, what's your use case here? I'm totally open to `createOrReturnNull` type behavior if it has to be done.

Reply:
I think a Try/catch is the best solution in this case.